src/library/scala/collection/immutable/Set.scala

```diff
@@ -78,6 +86,23 @@ object Set extends ImmutableSetFactory[Set] {
     def iterator: Iterator[Any] = Iterator.empty
     override def foreach[U](f: Any => U): Unit = ()
     override def toSet[B >: Any]: Set[B] = this.asInstanceOf[Set[B]]
+
+    override def ++[B >: Any, That](that: GenTraversableOnce[B])(implicit bf: CanBuildFrom[Set[Any], B, That]): That = {
```

Comment:
`SetLike` already covers the special cases. Does this override add anything new?

Reply:
ARn't these different methods?
`SetLike` doesn't have the `CanBuildFrom` implicit parameter

`  def ++ (elems: GenTraversableOnce[A]): This = {`

Reply:
Right. This old 2.12 stuff is confusing.